### ISSUE TITLE
Pretrained and finetuned model availability

### ISSUE THREAD

**Issue by jeavesj** [Alibaba-DAMO-DrugAI/MGNN#2]
Hello,

Thank you for your research! Are you willing to please post your pretrained and finetuned models for which you report best performance on the CASF-2016 test set please?

Thank you
